### PR TITLE
Add personal page and avatar for yanvictorsn.eth

### PR DESCRIPTION
**Pull request by YanVictorSN**
## Description

Add a personal page as a card with avatar, bio and social links.

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

 #8 

Your ENS/address: yanvictorsn.eth

### CODE REVIEW

File: packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { Address } from "~~/components/scaffold-eth";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <>
+      <div className="flex justify-center content-center pt-6">
+        <p className="mt-3 base-300 text-sm font-mono font-semibold"></p>
+        <div className="card w-80 bg-base-100 shadow-xl">
+          <figure className="px-10 pt-10">
+            <Image src={"/avatars/avatar-yan.png"} width={200} height={200} alt="yan's profile avatar on buildguild" />
```
Comment (carletex) [inline packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx:12]: You can remove the braces `{}` here

**Review by carletex — COMMENTED** [review]
Great stuff @YanVictorSN !!

Left a tiny comment.

Also, it seems that you are in Windows, right? You might be using CRLF as line endings (instead of LF, which we are using in the codebase). This is causing my prettier to reformat your page when saving it.

You can read more online about it: eg: https://stackoverflow.com/questions/2517190/how-do-i-force-git-to-use-lf-instead-of-crlf-under-windows

File: packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx (hunk 1 above)
Comment (YanVictorSN) [reply packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx]: Yes. I'm using windows. I didn't know that. I just changed it on my github. I also removed the braces from the image tag. Thanks for the review.

Comment (carletex) [reply packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx]: Great!

I'm still getting the CRLF line endings. Maybe you have to configure your git locally, re-save the file, and push?

I fixed it for you here: https://github.com/BuidlGuidl/batch0.buidlguidl.com/pull/13/commits/beb096c43b510d989828c0f60826e146dba1424d

Let's see if your settings are working on the next PR.

Merging this, thank you!!

Comment (YanVictorSN) [reply packages/nextjs/pages/builders/0xF51cFA5eb9128F064eafb75524502A039F063D91.tsx]: Thank you for the kindness. 

I configured it locally and cloned the repository again, I think everything is ok now.

**Pull request merged**